test/customer_test.rb

```diff
@@ -5,11 +5,11 @@
 class CustomerTest < MiniTest::Test
   attr_reader :parent, :customer
   def setup
-    sales_engine = SalesEngine.new
-    @parent = CustomerRepository.new('test/support/sample_customers.csv', sales_engine)
+    sales_engine = SalesEngine.new.startup
+    parent = CustomerRepository.new('test/support/sample_customers.csv', sales_engine)
```

Comment:
Line is too long. [86/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.